src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.civil.model;
```

Comment:
Merge master into your PR. This changes already exist in master.

Reply:
resolved